models/src_linkedin.yml

```diff
@@ -308,5 +308,78 @@ sources:
+          - name: conversion_value_in_local_currency
+            description: The value generated by your conversions, displayed in your LinkedIn account's local currency.
+          - name: cost_in_local_currency
+            description: The cost of the ads in the local currency.
+          - name: cost_in_usd
+            description: The cost of the ads in the USD.
+          - name: external_website_conversions
+            description: Number of conversions that occurred on an external website as a result of the ad.
+          - name: one_click_leads
+            description: Number of one-click leads generated from the ad campaign.
+          - name: external_website_conversions
+            description: The actions taken on your website that you've defined as valuable to your business after clicking on LinkedIn ads.
+          - name: one_click_leads
+            description: Leads submitted after clicking on LinkedIn ads.
+
+      - name: monthly_ad_analytics_by_member_region
+        description: Monthly LinkedIn Ads performance metrics aggregated by member region and campaign.
+        identifier: "{{ var('linkedin_ads_monthly_ad_analytics_by_member_region_identifier', 'monthly_ad_analytics_by_member_region') }}"
+        config:
+          enabled: "{{ var('ad_reporting__linkedin_ads_enabled', true) and var('linkedin_ads__using_monthly_ad_analytics_by_member_region', true) }}"
+        columns:
+          - name: campaign_id
+            description: Identifier of the LinkedIn Ads campaign.
+          - name: month
+            description: Month of the reporting data, aliased as `date_month`.
+          - name: clicks
+            description: The count of chargeable clicks.
+          - name: member_region
+            description: ID representing the region of the LinkedIn members who interacted with the ad.
+          - name: impressions
+            description: This is the count of "impressions" for Direct Ads and Sponsored Updates and "sends" for InMails.
+          - name: conversion_value_in_local_currency
+            description: The value generated by your conversions, displayed in your LinkedIn account's local currency.
+          - name: cost_in_local_currency
+            description: The cost of the ads in the local currency.
+          - name: cost_in_usd
+            description: The cost of the ads in the USD.
+          - name: external_website_conversions
+            description: Number of conversions that occurred on an external website as a result of the ad.
+          - name: one_click_leads
+            description: Number of one-click leads generated from the ad campaign.
+          - name: external_website_conversions
+            description: The actions taken on your website that you've defined as valuable to your business after clicking on LinkedIn ads.
           - name: one_click_leads
             description: Leads submitted after clicking on LinkedIn ads.
```

Comment:
I don't see these columns being brought in by the macro by default, should these be in here?

Reply:
Unsure how this happened. Removed.